ui/job-view/navbars/ActiveFilters.jsx

```diff
@@ -0,0 +1,161 @@
+            <span className="filtersbar-filter" key={filter.key}>
+              <span
+                className="pointable"
+                title={`Clear filter: ${filter.field}`}
+                onClick={() => this.thJobFilters.removeFilter(filter.key, filter.value)}
+              >
+                <i className="fa fa-times-circle" />&nbsp;
+              </span>
+              <span title={`Filter by ${filter.field}: ${filter.value}`}>
+                <b>{filter.field}:</b>
+                {filter.field === 'failure_classification_id' && (
+                  <span> {this.getFilterValue(filter.field, filter.value)}</span>
+                )}
+                {filter.field === 'author' && <span> {filter.value.split('@')[0].substr(0, 20)}</span>}
+                {filter.field !== 'author' && filter.field !== 'failure_classification_id' && <span> {filter.value.substr(0, 12)}</span>}
+              </span>
+            </span>))}
+          </div>}
+        {isFieldFilterVisible && <div>
+          <form className="form-inline">
+            <div className="form-group input-group-sm new-filter-input">
+              <label className="sr-only" htmlFor="job-filter-field">Field</label>
+              <select
+                id="job-filter-field"
+                className="form-control"
+                value={newFilterField}
+                onChange={evt => this.setNewFilterField(evt.target.value)}
+                placeholder="filter field"
+                required
+              >
+                <option value="" disabled>select filter field</option>
+                {Object.entries(this.fieldChoices).map(([field, obj]) => (
+                  obj.name !== 'tier' ? <option value={field} key={field}>{obj.name}</option> : null
+                ))}
+              </select>
+              <label className="sr-only" htmlFor="job-filter-value">Value</label>
+              {newFilterMatchType !== 'choice' && <input
+                className="form-control"
+                value={newFilterValue}
+                onChange={evt => this.setNewFilterValue(evt.target.value)}
+                id="job-filter-value"
+                type="text"
+                placeholder="enter filter value"
+              />}
+              <label className="sr-only" htmlFor="job-filter-choice-value">Value</label>
+              {newFilterMatchType === 'choice' && <select
+                className="form-control"
+                value={newFilterValue}
+                onChange={evt => this.setNewFilterValue(evt.target.value)}
+                id="job-filter-choice-value"
+              >
+                <option value="" disabled>select value</option>
+                {Object.entries(newFilterChoices).map(([fci, fci_obj]) => (
+                  <option value={fci} key={fci}>{fci_obj.name}</option>
+                )) }
+              </select>}
+              <button
+                className="btn btn-light-bordered btn-sm"
```

Comment:
Should this be given `type="button"` and the one below `type="reset"` (they otherwise default to submit):
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

A new eslint-plugin-react rule (that's going to be enabled in an upcoming eslint-config-airbnb) requires that a `type` be explicitly set:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/button-has-type.md

Reply:
That sounds good.  I'll add that.  Perhaps even ``submit`` for the ``add`` button since that causes the popup saying to set the values if they're not set.